src/canPowertrain.cpp

```diff
@@ -2,10 +2,36 @@
 
 CANPowertrain::CANPowertrain(CAN_TypeDef* canPort, CAN_PINS pins, int frequency) : CANManager(canPort, pins, frequency) {};
 
-void CANPowertrain::readHandler(CAN_message_t msg) {
+volatile bool mcu_battery_enable = false;
 
+void CANPowertrain::readHandler(CAN_message_t msg) {
+    uint8_t *data = msg.buf;
+    switch(msg.id){
+        case POWERTRAIN_BOOLS:
+            mcu_battery_enable = (data[0] >> 5) & 1;
+            break;
+        default:
+            break;
+    }
```

Comment:
This method actually doesn't need to be filled out right now since the Powertrain board isn't reading any CAN messages from the other boards. In particular, the MCU_BATT_EN is an output from the Nucleo that will be handled in IOManagement.